Doc/library/calendar.rst

```diff
@@ -426,17 +435,26 @@ The :mod:`calendar` module exports the following data attributes:
 
 .. data:: month_name
 
-   An array that represents the months of the year in the current locale.  This
+   A sequence that represents the months of the year in the current locale.  This
    follows normal convention of January being month number 1, so it has a length of
    13 and  ``month_name[0]`` is the empty string.
 
+       >>> import calendar
+       >>> list(calendar.month_name)
+       ['', 'January', 'February', 'March', 'April', 'May', 'June',
+       'July', 'August', 'September', 'October', 'November', 'December']
```

Comment:
This is failing doctest on the CI because of the newline:
```
Expected:
    ['', 'January', 'February', 'March', 'April', 'May', 'June',
    'July', 'August', 'September', 'October', 'November', 'December']
Got:
    ['', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
```

I think we can fix by either making it one long line:

```suggestion
       >>> list(calendar.month_name)
       ['', 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
```

Or using a [directive](https://docs.python.org/3/library/doctest.html#directives):

```suggestion
       >>> list(calendar.month_name)
       ['', 'January', 'February', 'March', 'April', 'May', 'June',  # doctest: +NORMALIZE_WHITESPACE
       'July', 'August', 'September', 'October', 'November', 'December']
```

I think a long line is okay here, it should be obvious what's scrolled off to the right.

Reply:
Agreed on the long line being okay.